package.json

```diff
@@ -3,7 +3,14 @@
   "version": "1.4.0",
   "description": "React Sample Applications for Okta",
   "scripts": {
-    "test": "eslint -c .eslintrc.json --ext .js --ext .jsx okta-hosted-login/src/ custom-login/src/"
+    "test": "eslint -c .eslintrc.json --ext .js --ext .jsx okta-hosted-login/src/ custom-login/src/ && npm run test:e2e",
+    "okta-hosted-login-server": "npm start --prefix okta-hosted-login/",
+    "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
+    "custom-login-server": "npm start --prefix custom-login/",
+    "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
+    "test:e2e": "npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
+    "get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git && git --git-dir=okta-oidc-tck/.git --work-tree=okta-oidc-tck checkout e2eTests",
```

Comment:
Also: we should add `okta-oidc-tck` to .gitignore, so that it's not creating noise in a local git status, and avoids accidentally adding it as a submodule.

Reply:
Done